hugegraph-core/src/main/java/com/baidu/hugegraph/backend/query/ConditionQuery.java

```diff
@@ -831,7 +831,7 @@ public boolean checkRangeIndex(HugeElement element, Condition cond) {
         }
 
         private static boolean removeFieldValue(Set<Object> values,
-                                                Object value){
+                                                Object value) {
```

Comment:
maybe we could merge them into one line

Reply:
if merge , this line is more than 80 characters.

https://github.com/apache/incubator-hugegraph/pull/1852#discussion_r859816036

Reply:
fine, we don't need to change the align of old code.

remember to rerun the failed ci